### DISCUSSION TITLE
How can I get 2 figures side-by-side in a pdf

### DISCUSSION THREAD

**Discussion by GEMcConnochie** [quarto-dev/quarto-cli#2772] [Q&A]
I am trying to get 2 figures in 2 columns in my quarto document. 
I have followed the example code

![image](https://user-images.githubusercontent.com/89376037/194786985-86449a26-78bb-4535-8143-90938ff71526.png)
 But the figures are not coming up next to each other in both the html and pdf. 

![image](https://user-images.githubusercontent.com/89376037/194787070-70056089-919c-467f-8e6f-60bc43f306c4.png)

How can I fix this?

Thanks

**Comment by rgaiacs** [comment]
Hi @GraceEMc, can you try

```
::: {#fig-feet layout="[45,45]"}
```

I remember have some issues with LaTeX when I use 50% + 50% as LaTeX will add some margin/padding around the image.
